### PR TITLE
AST-3717 - Forms UI Improvements Acceptance Feedback

### PR DESCRIPTION
**Pull request by neerajmasai**
### Description

JIRA - https://brainstormforce.atlassian.net/browse/AST-3717

### Screenshots

NA
### Types of changes

Improvements

### How has this been tested?

Test with the below popular plugins:
Contact form 7
WP forms
Ninja Forms Contact Form – The Drag and Drop Form Builder for WordPress
Forminator – Contact Form, Payment Form & Custom Form Builder
Contact Form Plugin – Fastest Contact Form Builder Plugin for WordPress by Fluent Forms
Gravity Forms
Formidable Forms – Contact Form, Survey, Quiz, Payment, Calculator Form & Custom Form Builder

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -5243,6 +5249,121 @@ public static function astra_sticky_sidebar_css() {
...
+					'font-size'     => '16px',
+					'font-style'    => 'normal',
+					'font-weight'   => '400',
+					'line-height'   => '24px',
+					'width'         => '100%',
+					'padding'       => '12px 16px',
+					'border-radius' => '4px',
+					'box-shadow'    => '0px 1px 2px 0px rgba(0, 0, 0, 0.05)',
+					'color'         => 'var(--ast-form-input-text, #475569)',
+				)
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:5338]: Add code inside the condition. If that plugin's constant is defined then only will load.

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -5243,6 +5249,121 @@ public static function astra_sticky_sidebar_css() {
...
+				),
+				'label, legend'      => array(
+					'color'       => '#111827',
+					'font-size'   => '14px',
+					'font-style'  => 'normal',
+					'font-weight' => '500',
+					'line-height' => '20px',
+				),
+				'fieldset'           => array(
+					'padding'       => '30px 30px 30px 30px',
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:5295]:
```suggestion
					'padding'       => '30px',
```

File: inc/class-astra-dynamic-css.php (hunk 1 above)
Comment (neerajmasai) [reply inc/class-astra-dynamic-css.php]: Done

File: inc/class-astra-dynamic-css.php (hunk 2 above)
Comment (neerajmasai) [reply inc/class-astra-dynamic-css.php]: Done

**Review by imnavanath — APPROVED** [review]

File: inc/core/builder/class-astra-builder-options.php [hunk 3]
```diff
@@ -957,6 +957,14 @@ function astra_hf_builder_customizer_defaults( $defaults ) {
...
 	 */
 	$update_colors_for_starter_library = Astra_Dynamic_CSS::astra_4_4_0_compatibility();
+	$update_color_for_forms_ui         = Astra_Dynamic_CSS::astra_forms_default_style_update();
+	$color_palette_7                   = '';
+	if ( $update_color_for_forms_ui ) {
+		$color_palette_7 = '#D1D5DB';
+	}
+	else {
+		$color_palette_7 = $update_colors_for_starter_library ? '#ADB6BE' : '#e2e8f0';
+	}
```
Comment (imnavanath) [inline inc/core/builder/class-astra-builder-options.php:967]:
```suggestion
	if ( $update_color_for_forms_ui ) {
		$color_palette_7 = '#D1D5DB';
	} else {
		$color_palette_7 = $update_colors_for_starter_library ? '#ADB6BE' : '#e2e8f0';
	}
```

**Review by imnavanath — APPROVED** [review]

**Pull request merged**